not panic return error

```go
package main

import (
	"fmt"

	clickhouse "github.com/AfterShip/clickhouse-sql-parser/parser"
)

func main() {
	query := "SELECT * FROM clickhouse where ( 1=1"
	parser := clickhouse.NewParser(query)
	// Parse query into AST
	statements, err := parser.ParseStatements()
	if err != nil {
		fmt.Println(err)
		return
	}

	fmt.Println(statements)
}
```
colx$ go run main.go
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x28 pc=0x493114]

goroutine 1 [running]:
github.com/AfterShip/clickhouse-sql-parser/parser.(*Parser).wrapError(0xc0000a1ee8, {0x4f7038, 0xc00008e0a0})
      /go/pkg/mod/github.com/!after!ship/clickhouse-sql-parser@v0.3.3/parser/parser_common.go:317 +0x494
github.com/AfterShip/clickhouse-sql-parser/parser.(*Parser).ParseStatements(0xc0000a1ee8)
       /go/pkg/mod/github.com/!after!ship/clickhouse-sql-parser@v0.3.3/parser/parser_table.go:985 +0x175
main.main()
     /code/learning/go/colx/main.go:13 +0x65
exit status 2

--- a/main.go
+++ b/main.go
@@ -57,7 +57,8 @@ func main() {
 	parser := clickhouse.NewParser(string(inputBytes))
 	stmts, err := parser.ParseStatements()
 	if err != nil {
-		panic(fmt.Sprintf("parse statements error: %s", err.Error()))
+		fmt.Printf("parse statements error: %s\n", err.Error())
+		os.Exit(1)
 	}
 	if !options.format { // print AST
 		bytes, _ := json.MarshalIndent(stmts, "", "  ") // nolint

--- a/parser/parser_common.go
+++ b/parser/parser_common.go
@@ -314,7 +314,11 @@ func (p *Parser) wrapError(err error) error {
 			for j := 0; j < column; j++ {
 				buf.WriteByte(' ')
 			}
-			buf.WriteString(strings.Repeat("^", len(p.lexer.lastToken.String)))
+			if p.last() != nil {
+				buf.WriteString(strings.Repeat("^", len(p.last().String)))
+			} else {
+				buf.WriteString("^")
+			}
 			buf.WriteByte('\n')
 		}
 	}

